### ISSUE TITLE
It would be very helpful if you could host a demo-site somewhere

### ISSUE THREAD

**Issue by SundialServices** [FlipperPA/peregrine#12]
I'm evaluating Django CMS options right now, and it would be very helpful to me if you had a running demo site somewhere.  ("PythonAnywhere.com," for example, could host such a thing for free.)  I just want to be able to quickly _see_ it in action . . .

**Comment by FlipperPA** [comment]
@SundialServices Thanks for the feedback! I'm going to talk to Divio about hosting one soon enough. In the meantime, Peregrine is built on top of Wagtail. If you want to spin up a version to check out, here's a link so you can see the interface you'll be getting:

https://control.divio.com/demo/get-new/wagtail/

Thank you!

**Comment by SundialServices** [comment]
Yes, thank you – I have seen the _WagTail_ bakery before.  But what I am specifically looking for is a _Peregrine_ demo site.  "Show me what _you_ got!" 😄
